CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Snowpark Python API Updates
 
+#### New Features
+- Added a new optional parameter called `format_json` to the `Session.SessionBuilder.app_name` function that allows to set the app name in the `Session.query_tag` in JSON format. By default, this parameter is set to `False`.
+
```

Comment:
can you add this into ### Improvements section

Reply:
Fixed on commit [249d3a7](https://github.com/snowflakedb/snowpark-python/pull/1845/commits/249d3a73bccea22b3cae1a0a428e247a87069ac8)